exercises/ansible_rhel/1.4-variables/README.md

```diff
@@ -172,7 +172,7 @@ For node3:
 
 ### Step 5 - Ansible Facts
 
-Ansible facts are variables that are automatically discovered by Ansible from a managed host. Remember the "Gathering Facts" task listed in the output of each `ansible-playbook` execution? At that moment the facts are gathered for each managed nodes. Facts can also be pulled by the `setup` module. They contain useful information stored into variables that administrators can reuse.
+Ansible facts are variables that are automatically discovered by Ansible from a managed host. Remember the "Gathering Facts" task listed in the output of each `ansible-navigation` execution? At that moment the facts are gathered for each managed nodes. Facts can also be pulled by the `setup` module. They contain useful information stored into variables that administrators can reuse.
```

Comment:
probably needs a new line to describe the scenario here, nothing called `ansible-navigation` exists

Reply:
@anshulbehl - that was just a typo on my end. should be navigator.